src/promptflow-devkit/promptflow/_cli/data/chat_flow/template/flow.dag.yaml.jinja2

```diff
@@ -14,17 +14,17 @@ outputs:
     is_chat_output: true
 nodes:
 - name: chat
-  type: llm
+  type: custom_llm
```

Comment:
feel risky to modify now, which may make local user fail to run new created flow.